src/current/v24.3/alter-virtual-cluster.md

```diff
@@ -70,10 +80,16 @@ You can use either:
 - `SYSTEM TIME` to specify a [timestamp]({% link {{ page.version.version }}/as-of-system-time.md %}).
 - `LATEST` to specify the most recent replicated timestamp.
 
+{{site.data.alerts.callout_info}}
+If you started the PCR stream with the `READ VIRTUAL CLUSTER` option, failing over with `SYSTEM TIME` will destroy the `readonly` virtual cluster. If you fail over with `LATEST`, the `readonly` virtual cluster will remain on the original standby cluster.
```

Comment:
Should we call out again that the readonly virtual cluster will not update with new writes after failover?

Reply:
good idea.

Reply:
Done, thanks!